spec/amber/cli/build_spec.cr

```diff
@@ -14,6 +14,7 @@ module Amber::CLI
       options = ["user:reference", "name:string", "body:text", "age:integer", "published:bool"]
       temp_options = options - ["user:reference", "age:integer"]
       MainCommand.run ["generate", "auth", "User"] | (options - ["user:reference"])
+      MainCommand.run ["generate", "error", "error"]
```

Comment:
I think, as is, you will have to give an argument (a name) which the generator will require, but also ignore.

All the other generators need a name. So, so we want to allow custom names, or figure out how to over-ride the requirement for this one?

Reply:
yeah, I think we need to overwrite the requirement for this one.  not sure how hard that is.

Reply:
Why is it `generate error error` instead of like `generate error 404` or something? error error seems strange semantically.